### ISSUE TITLE
TypeError: Cannot read property 'setupLayouts' of null

### ISSUE THREAD

**Issue by yaquawa** [frandiox/vitesse-ssr-template#2]
I just cloned this repo and ran `pnpm dev` got me this error:

```
TypeError: Cannot read property 'setupLayouts' of null
    at eval (/Users/xx/src/main.ts:20:38)
    at instantiateModule (/Users/xx/node_modules/.pnpm/vite@2.0.5/node_modules/vite/dist/node/chunks/dep-e0f09032.js:68041:166)
```

**Comment by frandiox** [comment]
@yaquawa  I couldn't reproduce this error in my environment. I've updated the dependencies in the repo to fix another issue in Vite 2.1.0. Can you try again with the updated deps?
Also, are you using Node >= 14?

**Comment by yaquawa** [comment]
Thanks @frandiox ! I could confirm that the latest version works!
